forLoopTimeout/forLoopTimeout.js

```diff
@@ -4,10 +4,14 @@
 // The output should be 1, 2, 3, .... 10. Right now it just prints 11.
 // I've been asked this three times in separate interviews.
 
+function setTimeoutFunk(i) {
+  setTimeout(function() {
+    console.log(i);
+  }, 0);
+}
+
 for (var i = 1; i <= 10; i++) {
-	setTimeout(function() {
+	setTimeoutFunk(i);
```

Comment:
There is some inconsistancy in your tabs here. Just make sure you pick a tab width and stick with it.